docs/get-involved/run-node/paratime-node.mdx

```diff
@@ -112,6 +112,40 @@ committee on Mainnet, your entity needs to:
 - Have a validator in the **validator set**.
 - Have at least **5,000,000.00 ROSE staked/delegated** to it.
 
+</TabItem>
+<TabItem value="Sapphire ParaTime">
```

Comment:
Should we make Sapphire the first tab?

Reply:
It's like you and @matevz are on the same page ❤️